### ISSUE TITLE
Mutifarm/crops balance discussion for 8.0

### ISSUE THREAD

**Issue by bedrill64** [GTNewHorizons/GT-New-Horizons-Modpack#4640]
Multifarm in manual orchard mode in its current state uses water and fertilizer only for harvesting, but crops remain fertilized and hydrated all the time, since its done while they are checked by the multifarm for maturity. It  leads to fact that crops that do not mature(GT ones) have permanent free environmental stats, thus higher growability, and they can still spread and crossbreed for no cost because the farm never charges for the water and fertilizer.

I mean, it applies to all crops, they get environmental stats basically for free, but there is usage of water/fert at harvesting stage. 

So, to balance free stats issue I have few ideas.
1) Change GT crops mechanic to be like barts, can't breed pre last stage, this would prevent ppl from using multifarm as breeding and spreading area. Multifarm would be completely useless for breeding since it would harvest, you would need crop-matrons to fertilize and water breeding areas or do it by hand. 
2) Give multifarm's scanning tool condition: if can't grow and cant harvest -> do not fertilize/hydrate, it would prevent crop from getting free stats if it can't grow, thus in some cases will cause crop to die due to lack of growability.
3) Change fertilizing / watering mechanic for multifarm's scanning tool. Make it require at least water each time it hydrates/fertilizes crop, usage of fertilizer in this case is bit hard to balance for me, so I'd leave it just with water consumption. It's hard because assuming that 100 nutrients lasts for 15 minutes, player would need 1000 fertilizer every 15 minutes just to fertilize his crops. This is hell high amount which can't be made without real fertilizer factory that uses pyros/distilation towers. It would make multifarm not worth a shot at all because how expensive it would be. Fertilizer is used for harvesting anyway. It would also make fertilia crop worthless since it would require more fertilizer to grow than it can really produce. Just too much.

Another worth mentioning point is that second autoharvesting tool available: harvester requires HV to be crafted and does not use fertilizer nor water to harvest stuff, so we can also leave it as it is as in case of multifarm both are used while harvesting.

Discussion open, lets balance 8.0.

**Comment by richardhendricks** [comment]
(edited the wording of Bedrill's comment a bit)

**Comment by richardhendricks** [comment]
1.  I like this alot, gives the Crop Matron a real use (if you want to wait for maturity before trying for a seedbag, or to try spreading the crop a bit before spading for a seedbag).

2. This would make me a little nervous since it could cause massive weed fields.  Would need extensive warning for players on this.

3. Are you assuming 1000 crops?  I mean....1000 crops *should* need alot of fertilizer!  Apatite + ashes are a totally viable source of fertilizer as well. 1000 fert in 15 minutes is only 1 stack/minute. 

The giant multilevel Multifarms players made using oreberries or XP berries are cute, but really IMO unbalancing and I think this is how to fix that.  Anyways, *most* crops you don't need running 24x7 continuously, once you stockpile a few hundred stacks you're good for a really long time, right?

I would also be OK with boosting fertilizer outputs from the various recipes *a bit* to help, not so much players don't need to setup systems.

**Comment by richardhendricks** [comment]
...maybe we should add a fertilizer or apatite bee?

**Comment by Anarack** [comment]
We have a fertilizer bee. "Ferterlizer" "apatite"

**Comment by Scottishprog** [comment]
I am currently NOT inclined to edit the multifarm re water/fertalizer use at this point. AT least, not without easing other sources of crop-supplied items. Or a better source of ashes/ easier to get Fertilia. If the water usage is boosted too high, the farm will not be usable until thirsty tanks/reservoirs. The Railcraft tanks are useless even for a small farm.

Keep in mind, the multifarm is one of the things that makes early steam usable.  (Steve's carts yields far too little for the cost.)

Noting the existence of bees is a _complete_ non-answer as far as I am concerned, and should not be used for balancing in there current form. They may as well not exist in there current form, due to the difficulty of breeding/using them without machines. (Available post HV or later.)

However, I _do_ agree with prevention of breeding on mutifarms. If you can do with without messing with breeding mechanic off of the multifarm, all the better.

**Comment by Scottishprog** [comment]
That being said,  if you can confine these changes just to the manual orchard mode, that might still be workable.

I would, however, _not_ make any changes that would bring the risk of weeds. I you plant on the field, and have no empty sticks, you should have a weed risk of zero.
Assuming the player is not spamming low level oreberries, a weed infestation could takes weeks/months to recover from, even if the player has leveled source plants on hand.  Seed bag chances are not 100%, along with slowish growth times make propagating plants a pain, especially for those below EV or so. (The crop manipulating/duplicating machines are not of much use until then. (For most useful crops)

Fertilia will need to be boosted so it's a net positive at base stats.

**Comment by richardhendricks** [comment]
This is only for IC2 crops, and only while scanning them to provide fert/water. 

It is simply ****not**** a given that players should have access to this level of extremely cheap crop breeding/growth at LV. Bart's Stirling water generators are new viable water generation option in LV, so water isn't a block IMO. And fertilizer as well. There simply is not enough *justification*  in my mind why an LV player should be able to freely grow massive multilayer farms with free speed acceleration for all crops.

We're looking at bees too, and they are *totally* applicable as a cornerstone of the modpack. It would be no different than saying we should balance around picks but not hammers. And I want to repeat, I am looking at bees and possible solutions.  An Apiary can be setup to provide partial automation of products and queens in LV, with frames to boost outputs, and the Alveary is available in MV to provide a massive boost in outputs and frame autiomation.

I disagree completely and 100% on your weed-free farms. If you want weed-free, the solution is simple - use crop matrons or weedX. Nothing preventing you from putting them down on top of your forestry farm!  Many mechanics in GT come with varying levels of danger - Boiler explosions, rain explosions, overvolt explosions, nuke explosions. Any of these can easily erase days/weeks of work.  Weeds are no different.  If you fail to take basic precautions - like separating different fields or having multiple isolated fields of the same crop, the onus is on you to keep an eye on it and not let it run ie overnight or while AFK. It would be no different than if the farm ran out of water/fertilizer/power.

Of course we would warn extensively before implementing such a change (if it even happens), similar to the bee warning right now in 7.x.

> Fertilia will need to be boosted so it's a net positive at base stats.
Net positive over what? Fertilizing itself? Fertilizing 10 other crops? We can discuss this, if it is even a crop we can change. That's one reason I like the bee idea better, since we have our own custom bees already implemented we can manipulate those stats easier.

**Comment by Scottishprog** [comment]
Per you first paragraph - Fair enough. I didn't really have a large farm until I got my biomass setup going. If the overall consumption of water/ferts was about the same as an auto tree farm for the same area and the same items/min output? (just a suggestion - this means a fermented biomass setup could sustain it.) So MV and later - assuming a reasonably accessible form of fertilizer from whatever source? Sure. As for crop breeding? In complete agreement.

Second paragraph - I should have made it plainer - I was speaking of breeding bees in the _current_ context. I see the long breeding trees, and no way to make it less aggravating at low levels. (I am not really a fan of the breeding mechanic. - personal preference.) I won't say anything further on this - you said you are looking into this, and I am happy to leave it at that.

I would agreed more on the weed risk if there were two issues:
 - The poisoning effect of weed-EX, even to children plants (FAR too harsh for an actual solution. We want to encourage people to go through the trouble of leveling stats, so fewer plants are needed.)
 - The time to recover from the loss of an entire farm worth of plants. (If propagation was a little faster, this might be workable)
In short, the risk might be ok, but I _don't_ agree with it being a long time to recover. (Having reserve plant to re-propagate from? This would be a recommended form of insurance.) That said, shouldn't be a short time either, or the weed risk doesn't really mean anything.
This is less so when EV+ World Accelerators are available. (I see this mostly as a HV and below issue, EV+ there are solutions available)

As for the Fertilia at base stats - 10%-25% above what is needed to fertilize it. (Barely net positive) For real yield, improve stats.

**Comment by richardhendricks** [comment]
There is no "child-crop" poisoning for Weed-Ex.  AFAIK, that is just a rumor and not confirmed in code. I think the confusion is that "Weed-Ex is applied to the Crop" - here Crop means Cropsticks.

Also, using Weed-Ex with Crop-Matrons is mentioned to not "overdose" the plants, so maybe using weed-ex with CMs doesn't cause stat damage.  Needs testing

**Comment by Scottishprog** [comment]
I think it's _very_ important that this get verified one way or the other.  If weed-ex via crop-matron is indeed weed prevention for the cost of weed-ex, then that greatly changes the situation. My understanding was that crop-matrons were very inefficient with the weed-ex. I would love to be proven wrong on both counts.

If I am indeed wrong, that makes this change more tolerable, and crop breeding more tolerable overall.
